homework/index.js

```diff
@@ -30,13 +30,108 @@
     return elem;
   }
 
+  function update(val, dBase) {
+    const container = document.getElementById('container');
+    const leftDiv = createAndAppend('div', container, { class: 'left-div whiteframe' });
+    const table = createAndAppend('table', leftDiv);
+    const tbody = createAndAppend('tbody', table);
+    const rep = dBase[val];
+
+    for (let ix = 0; ix < 4; ix++) {
+      const row = createAndAppend('tr', tbody);
+      const tdFirst = createAndAppend('td', row, { class: 'label' });
+      const tdSec = createAndAppend('td', row);
+      const updateDate = new Date(rep.updated_at).toLocaleString();
+      // eslint-disable-next-line default-case
+      switch (ix) {
+        case 0:
+          tdFirst.textContent = 'Repository :';
+          createAndAppend('a', tdSec, { text: rep.name, href: rep.html_url });
+          break;
+        case 1:
+          tdFirst.textContent = 'Description :';
+          tdSec.textContent = rep.description;
+          break;
+        case 2:
+          tdFirst.textContent = 'Forks :';
+          tdSec.textContent = rep.forks;
+          break;
+        case 3:
+          tdFirst.textContent = 'Updated :';
+          tdSec.textContent = updateDate;
+          break;
+      }
+    }
+
+    const rightDiv = createAndAppend('div', container, { class: 'right-div whiteframe' });
+    createAndAppend('p', rightDiv, {
+      text: 'Contributions',
+      class: 'contributor-header',
+    });
+    const ul = createAndAppend('ul', rightDiv, { class: 'contributor-list' });
+    const urlCol = rep.contributors_url;
+    fetchJSON(urlCol, (e, dt) => {
+      if (e) {
+        // Error message will be shown
```

Comment:
Not without some code to render it to the page.

Reply:
Sorry, I have forgotten to render errors in that line.